lib/response.js

```diff
@@ -0,0 +1,20 @@
+class Response {
+  constructor(resp) {
+    if (resp.constructor === Array) {
```

Comment:
You can use `if (Array.isArray(resp)) { }` to check for the same thing.

Reply:
Not doing that. - Thanks for the advice though! Next time I'll do that, but now we're getting rid of this file :D

Wouldn't be a problem if it was TypeScript... _flies away_